content/build/hydrobuild/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Hydrobuild
+description: Get started with Docker Hydrobuild
+keywords: build, cloud, cloud build, remote builder, hydrobuild
+---
+
+> **Early Access**
+>
+> Docker Hydrobuild is an early-access service that provides cloud-based
+> builders for your Docker organization.
+>
+> If you want to get involved in testing Hydrobuild, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{ .restricted }
+
+Hydrobuild is a service that lets you build your container images faster, both
+locally and in CI. Builds run on cloud infrastructure optimally dimensioned for
+your workloads, no configuration required. The service uses a remote build
+cache, ensuring fast builds anywhere and for all team members.
+
+## How Hydrobuild works
+
+Using Hydrobuild is no different from running a regular build. You invoke a
+build the same way you normally would, using `docker buildx build`. The
+difference is in where and how that build gets executed.
+
+By default when you invoke a build command, the build runs on a local instance
+of BuildKit, bundled with the Docker daemon. With Hydrobuild, you send the
+build request to a BuildKit instance running remotely, in the cloud.
+
+The remote builder executes the build steps, and sends the resulting build
+output to the destination that you specify. For example, back to your local
+Docker Engine image store, or to an image registry.
+
+Hydrobuild provides several benefits over local builds:
+
+- Improved build speed
+- Shared build cache
+- Native multi-platform builds
+
+And the best part: you don't need to worry about managing builders or
+infrastructure. Just connect to your builders, and start building.
+
+## Get Hydrobuild
+
+To get started with Hydrobuild, create a Docker account and sign up for the
+free plan at on the [Docker Cloud Build Dashboard](https://build.docker.com/).
```

Comment:
```suggestion
free plan on the [Docker Cloud Build Dashboard](https://build.docker.com/).
```

Reply:
I noticed when on staging that when I select the free plan, I'm asked to enter payment details. It may be worth mentioning this here